pkgs/top-level/all-packages.nix

```diff
@@ -5525,32 +5527,14 @@ in
 
   pinentry = callPackage ../tools/security/pinentry {
     libcap = if stdenv.isDarwin then null else libcap;
-    gcr = null;
-    qt4 = null;
-    qt5 = null;
-  };
-
-  pinentry_ncurses = res.pinentry.override {
-    gtk2 = null;
+    qt = qt5;
   };
 
-  pinentry_emacs = res.pinentry.override {
-    enableEmacs = true;
-  };
-
-  pinentry_gnome = res.pinentry.override {
-    inherit gcr;
-  };
-
-  pinentry_qt4 = res.pinentry.override {
-    gtk2 = null;
-    inherit qt4;
-  };
-
-  pinentry_qt5 = res.pinentry.override {
-    gtk2 = null;
-    inherit qt5;
-  };
+  pinentry-curses = pinentry.curses;
```

Comment:
Not sure if the dashes are really an improvement, but it certainly doesn't hurt to have both.

Reply:
We probably should change these to ```getOutput "curses"``` to be safer.

Reply:
Changed to use `getOutput`.

Reply:
> Not sure if the dashes are really an improvement, but it certainly doesn't hurt to have both.

The reasoning was that it will more closely match the binary names. I think we only had underlines here because Nix policy forbade dashes in package names. Since this is no longer the case, I found this a good opportunity to make this change.